src/assets/mockups/summaryData.jsx

```diff
@@ -1,54 +1,56 @@
 /** eslint verified */
 
 const areaData = {
-    id: 'LLA 96', // text
-    area: 101096, // number (hectareas)
-    description: 'La información espacial que se muestra en la figura corresponde a la identificación '
-      +'de condiciones sociales, culturales, económicas y biofísicas, algunas de las cuales son determinantes '
-      +'ambientales del ordenamiento territorial y pueden generar restricciones al desarrollo de las actividades '
-      +'de exploración y explotación de hidrocarburos. Este análisis hace parte del procedimiento que la ANH ha '
-      +'establecido para la coordinación y concurrencia con las entidades territoriales y demás autoridades y '
-      +'entidades con presencia en el territorio, con el fin de posibilitar la definición y determinación de '
-      +'nuevas áreas de interés de hidrocarburos.',
-    categories: [1, 7, 10], // TODO: Check categories with socioenvironmental record
+  id: 'LLA 96', // text
+  area: 101096, // number (hectareas)
+  description: 'La información espacial que se muestra en la figura corresponde a la identificación '
+    + 'de condiciones sociales, culturales, económicas y biofísicas, algunas de las cuales son determinantes '
+    + 'ambientales del ordenamiento territorial y pueden generar restricciones al desarrollo de las actividades '
+    + 'de exploración y explotación de hidrocarburos. Este análisis hace parte del procedimiento que la ANH ha '
+    + 'establecido para la coordinación y concurrencia con las entidades territoriales y demás autoridades y '
+    + 'entidades con presencia en el territorio, con el fin de posibilitar la definición y determinación de '
+    + 'nuevas áreas de interés de hidrocarburos.',
```

Comment:
Long strings should not be broken up: https://github.com/airbnb/javascript#strings--line-length

Also have this in mind: https://github.com/airbnb/javascript#es6-template-literals

Reply:
Solved in fec8146